### DISCUSSION TITLE
doubt

### DISCUSSION THREAD

**Discussion by ninjadoc107** [captcha-challenger/hcaptcha-model-factory#20] [Q&A]
![image](https://user-images.githubusercontent.com/110185860/187076085-8836d1be-36e8-45db-806e-866dbeb729af.png)

this is happening to me when i open the project

**Comment by yyyanbj** [comment]
Seems like you donot have any images in folder.

**Comment by ninjadoc107** [comment]
> Parece que você não tem nenhuma imagem na pasta.

what folder do i have to put the images in?

**Comment by yyyanbj** [comment]
Do you have trouble reading the wiki? Or you just do not know anything…

**Comment by ninjadoc107** [comment]
> 
I don't know anything, sorry

**Comment by yyyanbj** [comment]
Okay, don’t worry. I’ll update wiki and add a tutorial. But it’s my bedtime, I’ll do it later.

**Comment by ninjadoc107** [comment]
> Ok, não se preocupe. Vou atualizar o wiki e adicionar um tutorial. Mas é minha hora de dormir, farei isso mais tarde.

ok bro, good night, and thank you very much for your attention.

**Comment by yyyanbj** [comment]
Please check the tutorial here. https://github.com/beiyuouo/hcaptcha-model-factory/wiki#full-workflow-to-add-a-new-challenge

**Comment by ninjadoc107** [comment]
> Por favor, verifique o tutorial aqui. https://github.com/beiyuouo/hcaptcha-model-factory/wiki#full-workflow-to-add-a-new-challenge

ohh, now I understand almost everything, but the error of the images persists, I did as you said, changed the name of the task src/config.py, and then I created the folder data/smiling_dog/origin/yes
data/smiling_dog/origin/bad
and I put the photos with the dog smiling in yes and dogs not smiling in bad, but this error still persists

https://user-images.githubusercontent.com/110185860/187076085-8836d1be-36e8-45db-806e-866dbeb729af.png

**Comment by yyyanbj** [comment]
What is your running command? Do you add `--split`?

**Comment by ninjadoc107** [comment]
> 

yeah
python main.py --split --mode trainval

**Comment by yyyanbj** [comment]
Can you provide more information? such as file structure or something. I can not know anything in this screenshot.

**Comment by ninjadoc107** [comment]
> Voce pode fornecer mais informação? como estrutura de arquivos ou algo assim. Eu não posso saber nada nesta captura de tela.

you can leave it bro, it's giving you a lot of work

**Comment by yyyanbj** [comment]
Okay, if you have any question, feel free to ask.
